webapp-django/crashstats/crashstats/tests/test_models.py

```diff
@@ -335,6 +336,14 @@ def mocked_get(**params):
         assert r['Vendor'] == 'Mozilla'
         assert 'Email' in r  # no filtering at this level
 
+    def test_raw_crash_invalid_id(self):
+        model = models.RawCrash
+        model.implementation = self._mockeries[models.RawCrash]
+        api = model()
+
+        with pytest.raises(BadArgumentError):
+            api.get(crash_id='821fcd0c-d925-4900-85b6-687250180607docker/as_me.sh')
```

Comment:
This test is perfect! We might also want to verify that a request returns an HTTP 400 or something that's not an HTTP 500.

Alternatively (and I should have noticed this the first time I mentioned tests--sorry about that), we could add another test similar to this one that covers the "crash id is there, but not valid" case, but undoes the mocking:

https://github.com/mozilla-services/socorro/blob/b486727bf42f52327f3140ba4bcd35288df3868a/webapp-django/crashstats/api/tests/test_views.py#L376-L383